charmcraft.yaml

```diff
@@ -23,3 +23,39 @@ parts:
       - libssl-dev
       - rustc
       - cargo
+
+name: gateway-api-integrator
+title: Gateway API Integrator
+description: |
+  A [Juju](https://juju.is/) [charm](https://juju.is/docs/olm/charmed-operators) deploying and managing external access to HTTP/HTTPS services in a
+  Kubernetes cluster via a Gateway and an HTTPRoute resource. This requires the Kubernetes
+  cluster in question to have an Gateway API controller already deployed into it.
```

Comment:
I think a link to the docs for `Gateway API controller` would be in order here

Reply:
added the relevant link!